x-pack/plugins/uptime/server/lib/alerts/status_check.ts

```diff
@@ -19,18 +21,40 @@ import {
 } from '../../../common/runtime_types';
 import { MONITOR_STATUS } from '../../../common/constants/alerts';
 import { updateState, generateAlertMessage } from './common';
-import { commonMonitorStateI18, commonStateTranslations, DOWN_LABEL } from './translations';
+import {
+  commonMonitorStateI18,
+  commonStateTranslations,
+  statusCheckTranslations,
+} from './translations';
 import { stringifyKueries, combineFiltersAndUserSearch } from '../../../common/lib';
 import { GetMonitorAvailabilityResult } from '../requests/get_monitor_availability';
-import { GetMonitorStatusResult } from '../requests/get_monitor_status';
+import {
+  GetMonitorStatusResult,
+  GetMonitorDownStatusMessageParams,
+} from '../requests/get_monitor_status';
 import { UNNAMED_LOCATION } from '../../../common/constants';
 import { MonitorStatusTranslations } from '../../../common/translations';
 import { getUptimeIndexPattern, IndexPatternTitleAndFields } from '../requests/get_index_pattern';
 import { UMServerLibs, UptimeESClient, createUptimeESClient } from '../lib';
 import { ActionGroupIdsOf } from '../../../../alerting/common';
+momentDurationFormatSetup(moment);
 
 export type ActionGroupIds = ActionGroupIdsOf<typeof MONITOR_STATUS>;
 
+const getInterval = (timerangeCount: number, timerangeUnit: string): string => {
+  switch (timerangeUnit) {
+    case 's':
+      return moment.duration(timerangeCount, 'seconds').format('s [sec]');
+    case 'm':
+      return moment.duration(timerangeCount, 'minutes').format('m [min]');
+    case 'h':
+      return moment.duration(timerangeCount, 'hours').format('h [hr]');
+    case 'd':
+      return moment.duration(timerangeCount, 'days').format('d [day]');
+    default:
+      return `${timerangeCount} ${timerangeUnit}`;
+  }
+};
```

Comment:
nit: move this to separate file, this file is already bloated

Reply:
Yep, that's part of the small refactoring I started doing but haven't pushed yet.